controllers/presentationController.js

```diff
@@ -21,21 +21,37 @@ async function getAllPresentations(req, res) {
   }
 }
 
-// async function createPresentation(req, res, next) {
-//   // create
-// }
+async function createPresentation(req, res) {
+  const { title } = req.body;
 
-// async function savePresentation(req, res, next) {
-//   // save
-// }
+  try {
+    const newPresentation = await Presentation.create(title);
+
+    res.status(201).json(newPresentation);
+  } catch (err) {
+    res.status(500).send({ result: "error", error: "Internal Server Error" });
+  }
+}
+
+async function deletePresentation(req, res) {
+  const { id } = req.params;
+
+  try {
+    const deletedPresentation = await Presentation.findByIdAndDelete(id);
+
+    if (!deletedPresentation) {
+      res.status(404).send({ message: "No presentation found to delete" });
+      return;
+    }
 
-// async function deletePresentation(req, res, next) {
-//   // delete
-// }
+    res.status(200).send({ message: "Presentation successfully deleted" });
+  } catch (err) {
+    res.status(500).send({ result: "error", error: "Internal Server Error" });
+  }
+}
 
 module.exports = {
   getAllPresentations,
```

Comment:
```suggestion
  readAllPresentations,
```
createPresentation이 CRUD의 컨벤션을 따라 네이밍 되있으므로, getAllPresentations을 readAllPresentations로 통일시키는 게 좋을 것 같습니다.